phalcon/cache/backend/redis.zep

```diff
@@ -182,39 +192,26 @@ class Redis extends Backend implements BackendInterface
 	 */
 	public function save(keyName = null, content = null, lifetime = null, boolean stopBuffer = true)
 	{
-		var prefixedKey, lastKey, prefix, frontend, redis, cachedContent, preparedContent,
-			tmp, tt1, success, options, specialKey, isBuffering;
+		var redis, lastKey, frontend, cachedContent, preparedContent, ttl, success, specialKey, isBuffering;
+
+		let redis = this->_getRedis();
 
-		if keyName === null {
+		if !keyName {
```

Comment:
If user passed empty string or `0`... An empty string is not allowed but `0` should be allowed.

I think the check should be

```zephir
		if keyName === null || keyName === "" {
			let lastKey = this->_lastKey;
		} else {
			let lastKey = this->_prefix . keyName;
			let this->_lastKey = lastKey;
		}
```